docs(pattern): import pattern (#199)
carbon-design-system/carbon-website: src/pages/experimental/import-pattern/index.mdx

---
title: Import pattern
---

<PageDescription>

The import action transfers data or objects from an external source into a system.

</PageDescription>

#### Status:

[Experimental](experimental/about)

#### Maintainers:

[Vikki Paterson](https://github.com/vikkipaterson)

![Example of an import modal in context](/images/0.5.png)

<Caption>Example of an import modal in context</Caption>

## Choose a file

> Wherever possible there should always be a drag and drop zone. Use a file drop component, and offer the option to browse locally and select a file. Note that file selection should be restricted to allowed file types.

Reviewer: ```suggestion
Wherever possible there should be a drag and drop zone. Use a file drop component and offer the option to browse locally to select a file. Note that file selection should be restricted to allowed file types.
```

<Row>
<Column colLg={8}>

> ![Example of an import dialog](/images/2.png)

Author: ```suggestion
![Example of an import modal](/images/2.png)
```

> ![Example of an import dialog](/images/2.png)

Reviewer: ```suggestion
![Example of an import modal](/images/2.png)
```

> ![Example of an import dialog](/images/2.png)

Author: ```suggestion
![Example of an import modal](/images/2.png)
```

> ![Example of an import dialog](/images/2.png)

Reviewer: ```suggestion
![Example of an import modal](/images/2.png)
```

---
title: Import pattern
---

<PageDescription>

> Importing a resource transfers data or objects into a system from an external source.

Reviewer: ```suggestion
The import action transfers data or objects from an external source into a system.
```

</PageDescription>

#### Status:

[Experimental](experimental/about)

#### Maintainers:

[Vikki Paterson](https://github.com/vikkipaterson)

> <AnchorLinks>

Reviewer: ```suggestion

```

> <AnchorLink>Choose a file</AnchorLink>

Reviewer: ```suggestion
```

> <AnchorLink>Enter a URL</AnchorLink>

Reviewer: ```suggestion

```

> </AnchorLinks>

Reviewer: ```suggestion

```
